UIKit+Combine/UICollectionView+Combine.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  UICollectionView+Combine.swift
```

Comment:
This is in the wrong folder, should be in **Sources/Controls** and no need for the UIKit+Combine folder